database/sample_data/bookings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = [
```

Comment:
this file seems pretty small. Does it need its own file? idk where it's used so maybe there is a reason